homework/index.js

```diff
@@ -0,0 +1,30 @@
+const promise = fetch('https://api.github.com/orgs/HackYourFuture/repos?per_page=100')
+  .then(response => response.json());
+
+function selectRepo() {
```

Comment:
Actually, I'd rather call this function 'populateRepoList' or something; to more reflect what it is actually doing.